api_core/setup.py

```diff
@@ -40,7 +40,7 @@
 ]
 extras = {
     'grpc': 'grpcio >= 1.8.2',
-    'grpcio-gcp': 'grpcio-gcp >= 0.2.2'
+    'grpcgcp': 'grpcio-gcp >= 0.2.2'
```

Comment:
I could optionally keep the old one around just in case anyone took a reliance on the name, but was avoiding cruft since the google-cloud-* packages should be the only packages depending on this, and spanner uses this only.

Reply:
Also, *technically* the time between api_core and spanner going out this extra won't exist. I will tag/release at the same time so this should be a small timeframe